src/icons/minus.jsx

```diff
@@ -0,0 +1,16 @@
+export default ({ width, height }) => {
+  return (
+    <svg
+      width={width || '25'}
```

Comment:
Should this be 24 as well? Viewbox is square

Reply:
Done